config.toml

```diff
@@ -32,3 +32,9 @@ download = "/tmp/vimana/gmd/rollup_mocha.sh"
 # [components.berachain.full]
 # binary = "/path/to/celestia/binary"
 # download = "https://example.com/berachain.zip"
+
+[components.eigen]
```

Comment:
please comment this block out as we are yet to be done with the operator running service.

Reply:
ok